internal/pkg/provider/resources/machine.go

```diff
@@ -9,9 +9,10 @@ import (
 	"github.com/cosi-project/runtime/pkg/resource/meta"
 	"github.com/cosi-project/runtime/pkg/resource/protobuf"
 	"github.com/cosi-project/runtime/pkg/resource/typed"
+	"github.com/siderolabs/omni/client/pkg/infra"
 
 	"github.com/siderolabs/talemu/api/specs"
-	"github.com/siderolabs/talemu/internal/pkg/machine/runtime/resources/emu"
+	providerMeta "github.com/siderolabs/talemu/internal/pkg/provider/meta"
```

Comment:
```suggestion
	providermeta "github.com/siderolabs/talemu/internal/pkg/provider/meta"
```

I think naming conventions would suggest using all lowercase here